src/chromyRunner.test.js

```diff
@@ -69,7 +69,6 @@ describe('ChromyRunner', () => {
       expect(loggerCalls[0]).toEqual('goto -> www.example.com');
       expect(loggerCalls[1]).toEqual('Capturing screenshot of whole DOM');
       expect(loggerCalls[2]).toEqual('screenshot saved in -> ./differencify_report/default.png');
-      expect(loggerCalls[4]).toEqual('Saving the diff image to disk');
```

Comment:
Totally agree that relying on verifying logs is not enough to validate your test, but it is a good safe guard beside other functionalities IMO